jekyll/_cci2/server-3-operator-nomad.adoc

```diff
@@ -0,0 +1,118 @@
+---
+version:
+- Server v3.x
+- Server Admin
+---
+= Introduction to Nomad Cluster Operation
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Learn how to operate the Nomad Cluster in your CircleCI 2.0 installation.
+:icons: font
+:toc: macro
+:toc-title:
+
+CircleCI uses https://www.hashicorp.com/blog/nomad-announcement/[Nomad] as the primary job scheduler. This chapter provides a basic introduction to Nomad for understanding how to operate the Nomad Cluster in your CircleCI installation.
+
+toc::[]
+
+== Basic Terminology and Architecture
+
+.Nomad Cluster Management
+image::nomad-diagram-v2.png[Diagram of the Nomad cluster]
+<<<
+- **Nomad Server:** Nomad servers are the brains of the cluster; they receive and allocate jobs to Nomad clients. In CircleCI, a Nomad server runs on your Services machine as a Docker Container.
+
+- **Nomad Client:** Nomad clients execute the jobs they are allocated by Nomad servers. Usually a Nomad client runs on a dedicated machine (often a VM) in order to fully take the advantage of machine power. You can have multiple Nomad clients to form a cluster and the Nomad server allocates jobs to the cluster with its scheduling algorithm.
+
+- **Nomad Jobs:** A Nomad job is a specification, provided by a user, that declares a workload for Nomad. A Nomad job corresponds to an execution of a CircleCI job. If the job uses parallelism, say 10 parallelism, then Nomad will run 10 jobs.
+
+- **Build Agent:** Build Agent is a Go program written by CircleCI that executes steps in a job and reports the results. Build Agent is executed as the main process inside a Nomad Job.
+
+== Basic Operations
+
+The following section is a basic guide to operating a Nomad cluster in your installation.
+
+The `nomad` CLI is installed in the Nomad pod. It is pre-configured to talk to the Nomad cluster, so it is possible to use the `nomad` command to run the following commands in this section.
+
+=== Checking the Jobs Status
+
+The get a list of statuses for all jobs in your cluster, run:
+
+```shell
+kubectl exec -it nomad-pod-IDm -- nomad status
+```
+
+The `Status` is the most important field in the output, with the following status type definitions:
+
+- `running`: Nomad has started executing the job. This typically means your job in CircleCI is started.
+
+- `pending`: There are not enough resources available to execute the job inside the cluster.
+
+- `dead`: Nomad has finished executing the job. The status becomes `dead` regardless of whether the corresponding CircleCI job/build succeeds or fails.
+
+=== Checking the Cluster Status
+
+To get a list of your Nomad clients, run:
+
+```shell
+kubectl exec -it nomad-pod-IDm -- nomad node-status
```

Comment:
```suggestion
kubectl exec -it nomad-server-pod-ID -- nomad node-status
```

Reply:
Actually I see IDm referenced all over this doc. What is the m?

Reply:
@ryanwohara It's from here: https://circleci.slack.com/archives/GKZGNMNR5/p1646750514610919?thread_ts=1646750305.583159&cid=GKZGNMNR5
maybe it was a typo?

Reply:
I think it was a typo. I'll make more suggestions so it's easy to resolve.

Reply:
When copy and paste goes bad! Thank you 🙏